examples/jsm/nodes/misc/TextureCubeUVNode.js

```diff
@@ -165,17 +167,92 @@ TextureCubeUVNode.prototype = Object.create( TempNode.prototype );
 TextureCubeUVNode.prototype.constructor = TextureCubeUVNode;
 TextureCubeUVNode.prototype.nodeType = "TextureCubeUV";
 
+TextureCubeUVNode.prototype.bilinearCubeUV = function ( builder, texture, uv, mipInt ) {
+	
+	var bilinearCubeUV = new FunctionCallNode( TextureCubeUVNode.Nodes.bilinearCubeUV, [ texture, uv, mipInt ] );
+	
+	this.colorSpaceTL = this.colorSpaceTL || new ColorSpaceNode( new ExpressionNode( '', 'v4' ) );
+	this.colorSpaceTL.fromDecoding( builder.getTextureEncodingFromMap( this.value.value ) );
+	this.colorSpaceTL.input.parse( bilinearCubeUV.build( builder ) + '.tl' );
+	
+	this.colorSpaceTR = this.colorSpaceTR || new ColorSpaceNode( new ExpressionNode( '', 'v4' ) );
+	this.colorSpaceTR.fromDecoding( builder.getTextureEncodingFromMap( this.value.value ) );
+	this.colorSpaceTR.input.parse( bilinearCubeUV.build( builder ) + '.tr' );
+	
+	this.colorSpaceBL = this.colorSpaceBL || new ColorSpaceNode( new ExpressionNode( '', 'v4' ) );
+	this.colorSpaceBL.fromDecoding( builder.getTextureEncodingFromMap( this.value.value ) );
+	this.colorSpaceBL.input.parse( bilinearCubeUV.build( builder ) + '.bl' );
+	
+	this.colorSpaceBR = this.colorSpaceBR || new ColorSpaceNode( new ExpressionNode( '', 'v4' ) );
+	this.colorSpaceBR.fromDecoding( builder.getTextureEncodingFromMap( this.value.value ) );
+	this.colorSpaceBR.input.parse( bilinearCubeUV.build( builder ) + '.br' );
+	
+	var f = bilinearCubeUV.build( builder ) + '.f';
```

Comment:
It seems variable `f` is not used.

Reply:
Yes, is more simple remove without create other PR?

Reply:
see b78e8c431c7f90c28b14a580f073f675bcb30842 :+1:

Reply:
Thanks!